charts/stable/quassel-core/values.yaml

```diff
@@ -11,6 +11,14 @@ podSecurityContext:
   runAsUser: 0
   runAsGroup: 0
 
+probes:
+  liveness:
+    enabled: false
+  readiness:
+    enabled: false
+  startup:
+    enabled: false
```

Comment:
Wait, this is not good.
Probes are there to test if the backend is up or not.

Reply:
I'll make it work first then i'll figure out where the probes should probe ;)

Reply:
You can use a TCP or UDP probe on just like any port ;-)

Reply:
In this case the default would be a TCP probe on `4242`

Reply:
IIRC, TCP is the default. 99% wasn't the probes the issue but just to rule it out.